packages/components/src/styles/typography.css

```diff
@@ -0,0 +1,252 @@
+body {
+  font-family: metropolis, sans-serif !important;
```

Comment:
We should try to avoid at all cost to have `!important`s in the basic styles we provide. This can be left for later to change, it is just something that has a high risk of being forgotten here :)

Reply:
by all means, this is still copy/paste old Quanta thinggies that we should polish.